README.rst

```diff
@@ -107,8 +107,11 @@ and see the created barcode. That's it.
 
 Commandline::
 
-    $ python-barcode create "My Text" outfile
+    `$ python-barcode create "123456789000" outfile -b ean --text "text to appear under barcode" `
     New barcode saved as outfile.svg.
+    
+    # This may not work if `ImageWriter` is None
+    # ImageWriter is unavailable if Pillow is not installed (it's required for exporting to images instead of SVG).
```

Comment:
This is just too specific. You're talking about a variable being `None` in the readme, and there's no context to where this variable is defined (end users shouldn't really even care about that).

Also, it will *never* work, so it's not `may not work`. :wink: 

I'd go with something like:
```
The following will not work in Pillow is not installed (Pillow is required for exporting to images instead of SVG).
```

Reply:
I was thinking the same thing :smiley:  but wasn't sure what to say